lib/core/src/debug/monitor.rs

```diff
@@ -587,7 +640,7 @@ impl Sampler for Fps {
         if self.begin_time > 0.0 {
             let end_time     = time;
             self.value       = 1000.0 / (end_time - self.begin_time);
-            self.value_check = self.check_by_treshold(55.0,25.0);
+            self.value_check = self.check_by_threshold(55.0,25.0);
```

Comment:
So, I think the 55.0 and 25.0 should be a constants, and in the previous place use 1000.0/FPS_WARN_THRESHOLD, and 1000.0/FPS_ERROR_THRESHOLD

Reply:
I was not up to date. But maybe the constants FPS_TIME_WARN_THRESHOLD should use FPS_WARN_THRESHOLD instead of hardcoded 55.0?